tests/foreman/cli/test_docker.py

```diff
@@ -1320,3 +1320,65 @@ def test_positive_remove_docker_repo_ccv(
         )
         activation_key = module_target_sat.cli.ActivationKey.info({'id': activation_key['id']})
         assert activation_key['content-view'] != comp_content_view['name']
+
+
+class TestPodman:
+    """Tests specific to using podman push/pull on Satellite
+
+    :CaseComponent: Podman
+
+    :team: Phoenix-content
+    """
+
+    @pytest.fixture(scope='class')
+    def enable_podman(module_product, module_target_sat):
+        """Enable base_os and appstream repos on the sat through cdn registration and install podman.
+        Also includes a change to the katello.yaml file, and a restart of foreman"""
+        module_target_sat.register_to_cdn()
+        if module_target_sat.os_version.major > 7:
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhel_bos']['id'])
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhel_aps']['id'])
+        else:
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhscl']['id'])
+            module_target_sat.enable_repo(module_target_sat.REPOS['rhel']['id'])
+        result = module_target_sat.execute(
+            'dnf install -y --disableplugin=foreman-protector podman'
+        )
+        assert result.status == 0
+        result = module_target_sat.execute(
+            r"sed -i '/:katello:/a\ \ :container_image_registry:\n    :allow_push: true\n' /etc/foreman/plugins/katello.yaml"
+        )
```

Comment:
Is this gonna be handled by installer?

Reply:
Eventually, I think so, yes. My understanding though, is for the first iteration of this feature it will have to be manually enabled.